qubesmanager/qube_manager.py

```diff
@@ -1352,6 +1372,18 @@ def table_selection_changed(self):
             if not vm.updateable and vm.klass != 'AdminVM':
                 self.action_updatevm.setEnabled(False)
 
+            if vm.state['power'] == 'Halted'and \
+                    vm.vm.features.get('prohibit-start', False):
```

Comment:
```suggestion
            if vm.state['outdated'] == 'blocked':
```
Don't use direct API calls here, that's what the whole VmInfo object is for.

Reply:
Done